src/probeinterface/io.py

```diff
@@ -965,6 +965,26 @@ def write_csv(file, probe):
             "ap_hp_filters",
         ),
     },
+    # NP-Opto
+    "opto": {
```

Comment:
Maybe opto will not be futur proff as a name, they could release several "opto" probe no ?
Is this naming official ?